community-code/HOW_TO_BUILD_A_RAG_SYSTEM_USING_LANGCHAIN/notebooks/run.ipynb

```diff
@@ -0,0 +1,1961 @@
+       "\n",
+       "    factual_correctness  faithfulness  semantic_similarity  \\\n",
+       "36                 0.40      0.750000             0.929504   \n",
+       "37                  NaN      0.000000             0.739923   \n",
+       "38                 0.80      0.125000             0.979062   \n",
+       "39                 0.29      0.100000             0.941395   \n",
+       "40                 0.17      0.428571             0.907696   \n",
+       "\n",
+       "    noise_sensitivity_relevant  \n",
+       "36                        0.25  \n",
+       "37                        0.00  \n",
+       "38                        0.00  \n",
+       "39                        0.00  \n",
+       "40                        0.00  "
+      ]
+     },
+     "execution_count": 69,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df.tail()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 70,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df.to_csv(\"eval_results.csv\", index=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "vscode": {
+     "languageId": "sql"
+    }
+   },
+   "source": [
+    "## Neptune Experimental Tracking"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 71,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "os.environ[\"NEPTUNE_PROJECT\"] = \"community/building-RAG-using-LangChain\"\n",
+    "os.environ[\"NEPTUNE_API_TOKEN\"] = \"eyJhcGlfYWRkcmVzcyI6Imh0dHBzOi8vYXBwLm5lcHR1bmUuYWkiLCJhcGlfdXJsIjoiaHR0cHM6Ly9hcHAubmVwdHVuZS5haSIsImFwaV9rZXkiOiIzZDQ4YmZmMS1jNzA0LTRhMDEtYTIxMi1lNTg4ZWQ1MjliNjMifQ==\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 72,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "[neptune] [warning] NeptuneWarning: By default, these monitoring options are disabled in interactive sessions: 'capture_stdout', 'capture_stderr', 'capture_traceback', 'capture_hardware_metrics'. You can set them to 'True' when initializing the run and the monitoring will continue until you call run.stop() or the kernel stops. NOTE: To track the source files, pass their paths to the 'source_code' argument. For help, see: https://docs.neptune.ai/logging/source_code/\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[neptune] [info   ] Neptune initialized. Open in the app: https://app.neptune.ai/community/building-RAG-using-LangChain/e/BUIL1-5\n"
+     ]
+    }
+   ],
+   "source": [
+    "import neptune\n",
+    "\n",
+    "run = neptune.init_run()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 73,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Track the eval file\n",
+    "\n",
+    "run[\"eval_data_versions\"].track_files(\"eval_results.csv\")"
```

Comment:
Is there any particular reason for using `track_files()` instead of `upload()`?

Reply:
No. Will update to `upload()` instead.